app/controllers/api/automation_requests_controller.rb

```diff
@@ -41,5 +41,16 @@ def deny_resource(type, id, data)
     rescue => err
       action_result(false, err.to_s)
     end
+
+    def find_automation_requests(id)
```

Comment:
It might be good to do this as a follow up since there might be more opportunities to do this outside the scope of fixing these two endpoints